RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h

```diff
@@ -2,11 +2,9 @@
 #include "DataFormats/VertexReco/interface/Vertex.h"
 #include <limits>
 
-namespace {
```

Comment:
@Dr15Jones 
this file has "using X = Y;"
static analyzer does not report it as an issue.
Should it be?